dagql/idtui/frontend.go

```diff
@@ -32,6 +33,12 @@ type FrontendOpts struct {
 
 	// Verbosity is the level of detail to show in the TUI.
 	Verbosity int
+
+	// Don't show things that completed beneath this duration. (default 100ms)
+	TooFastThreshold time.Duration
```

Comment:
I like this, will save a lot of logs, however will this still appears on cloud?

Reply:
Yeah this is only for the local TUI. I've experimented with a similar change for Cloud but haven't found a version I'm 100% happy with yet.